apps/core/scenes/passengerForm/BaggageBundles/BaggageTypeDetail.js

```diff
@@ -0,0 +1,115 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { StyleSheet, Icon } from '@kiwicom/universal-components';
+import type { IconNameType } from '@kiwicom/universal-components';
+import { Text } from '@kiwicom/margarita-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import { bags, type Bag } from './BagTypes';
+
+type Props = {|
+  +bagType: Bag,
+  +dimensions?: string,
+  +quantity: number,
+  +chosen: boolean,
+  +weight: string,
+|};
+
+function getIcon(bagType, quantity: number): IconNameType {
+  if (quantity === 0 && bagType === bags.PERSONAL_ITEM)
+    return 'baggage-personal-item'; // @TODO add "no personal item icon"
+  if (quantity === 0 && bagType === bags.CHECKED_BAG) return 'close';
```

Comment:
You can save one duplicity condition here :)

```js
if (quantity === 0) {
  if (bagType === bags.PERSONAL_ITEM)
    return 'baggage-personal-item'; // @TODO add "no personal item icon"
  if (bagType === bags.CHECKED_BAG) 
    return 'close';
}
```

Reply:
Good point! I ll change it